dao/OrderDao.py

```
@@ -0,0 +1,267 @@
+            if 'items' in order_data and order_data['items']:
+                item_sql = sql.SQL("""
+                    INSERT INTO order_items (
+                        order_id,
+                        product_id,
+                        product_name,
+                        quantity,
+                        price,
+                        subtotal
+                    ) VALUES (%s, %s, %s, %s, %s, %s)
+                """)
+                
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)